src/waypoint.jsx

```diff
@@ -184,6 +198,62 @@ export default class Waypoint extends React.Component {
   }
 
   /**
+   * @param {string|number} offset Blah blah blah
+   * @param {number} contextHeight Blah blah blah
```

Comment:
lol, you might want to change this description a little. I think just removing the blah-blah is fine too.